src/libsync/configfile.cpp

```diff
@@ -113,6 +116,13 @@ ConfigFile::ConfigFile()
 
     QSettings settings(config, QSettings::IniFormat);
     settings.beginGroup(defaultConnection());
+
+    // run init only once
+    static bool init = [this]() {
+        setLogHttp(logHttp());
+        return false;
+    }();
+    Q_UNUSED(init);
```

Comment:
I really dislike this singleton-ish behavior. 

Can't you call this explicitly from main.cpp or from application or something?

Reply:
Well yes with 2.8 I will rework the whole class.
For now I'd like to just do this...